new/resource-current-value.md

```diff
@@ -0,0 +1,98 @@
+---
+RFC: unassigned
+Author: John Keiser <jkeiser@chef.io
+Status: Draft
+Type: Standards Track
+---
+
+# Allow Current Value To Be Read From Resource
```

Comment:
But you left in `current_value` when the examples further down call the new DSL `current_resource`, right? Or is this proposing both?